api/v1_projects.inc

```diff
@@ -316,3 +317,86 @@ function api_v1_projects_pagerounds($method, $data, $query_params)
 
     return array_merge(["OCR"], array_keys($Round_for_round_id_));
 }
+
+//---------------------------------------------------------------------------
+// projects/charsuites
+
+function api_v1_projects_charsuites($method, $data, $query_params)
+{
+    if (isset($query_params["enabled"])) {
+        $charsuites = CharSuites::get_enabled();
+    } else {
+        $charsuites = CharSuites::get_all();
+    }
+
+    $return_data = [];
+    foreach ($charsuites as $charsuite) {
+        $return_data[] = [
+            "id" => $charsuite->name,
+            "name" => $charsuite->title,
+            "characters" => convert_codepoint_ranges_to_characters($charsuite->codepoints),
+            "enabled" => $charsuite->is_enabled(),
+        ];
+    }
+
+    return $return_data;
+}
+
+//---------------------------------------------------------------------------
+// projects/specialdays
+
+function api_v1_projects_specialdays($method, $data, $query_params)
+{
+    $return_data = [];
+
+    $special_days = load_special_days();
+    foreach ($special_days as $spec_code => $special_day) {
+        if (isset($query_params["enabled"]) && !($special_day["enable"] == 1)) {
+            continue;
+        }
+        $return_data[] = [
+            "id" => $special_day["spec_code"],
+            "name" => $special_day["display_name"],
+            "comment" => $special_day["comment"],
+            "url" => $special_day["info_url"],
+            "color" => $special_day["color"],
+            "symbol" => $special_day["symbol"],
+            "date_open" => sprintf("%02d-%02d", $special_day["open_month"], $special_day["open_day"]),
```

Comment:
do we ever care about time zones or are these days always UTC and no concern with time zones in the "future"

Reply:
DP time is all "server time", so it isn't UTC and we also don't need to worry about timezones. It's an interesting artifact from the initial developers.

Reply:
See the comment [here](https://github.com/DistributedProofreaders/dproofreaders/blob/d01225fed2fddcad30ab069a46d0a9d355293175/crontab/onoff_special_event_queues.php#L24). That applies only to defined special days, though, not birthdays and otherdays.

Today's been busier than I expected -- I haven't done any reviewing or testing, yet, but wanted to go ahead and point to the time zone info. I don't know if that's relevant to the API, though.